specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/stable/2019-03-01/AlertsManagement.json

```diff
@@ -1388,6 +1241,34 @@
         }
       }
     },
+    "alertsMetaData": {
+      "description": "alert meta data information.",
+      "properties": {
+        "properties": {
```

Comment:
Each "alertMetadata" resource should look like a resource with a name, type, and ID.  They should also have individual GETs if you are modeling them as a GETable resource.  Basically, make the API and definitions look a lot more like /alerts

Reply:
@pilor this change is only documenting the current API that our RP exposes. Would it be OK to approve this change, and we'll open a bug to fix the API in the coming sprints?

Reply:
Any new api-versions for this API need to conform to the RPC.  You won't be able to make changes here in the future without implementing that